packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -108,44 +104,33 @@ describe("<DateRangeInput>", () => {
             // callbacks after responding to these events ourselves within the component.)
 
             it("fires custom onChange callback", () => {
-                assertEventCallbackFired("change", (spy) => {
-                    return { onChange: spy };
-                });
+                assertEventCallbackFired("change", (spy) => ({ onChange: spy }));
             });
 
             it("fires custom onFocus callback", () => {
-                assertEventCallbackFired("focus", (spy) => {
-                    return { onFocus: spy };
-                });
+                assertEventCallbackFired("focus", (spy) => ({ onFocus: spy }));
             });
 
             it("fires custom onBlur callback", () => {
-                assertEventCallbackFired("blur", (spy) => {
-                    return { onBlur: spy };
-                });
+                assertEventCallbackFired("blur", (spy) => ({ onBlur: spy }));
             });
 
             it("fires custom onClick callback", () => {
-                assertEventCallbackFired("click", (spy) => {
-                    return { onClick: spy };
-                });
+                assertEventCallbackFired("click", (spy) => ({ onClick: spy }));
             });
 
             it("fires custom onKeyDown callback", () => {
-                assertEventCallbackFired("keydown", (spy) => {
-                    return { onKeyDown: spy };
-                });
+                assertEventCallbackFired("keydown", (spy) => ({ onKeyDown: spy }));
             });
 
             it("fires custom onMouseDown callback", () => {
-                assertEventCallbackFired("mousedown", (spy) => {
-                    return { onMouseDown: spy };
-                });
+                assertEventCallbackFired("mousedown", (spy) => ({ onMouseDown: spy }));
```

Comment:
you wanna get real clever, you name the param the same as key. 
`(onMouseDown) => ({ onMouseDown })`